tests/rustdoc-js/doc-alias-after-other-items.js

```diff
@@ -0,0 +1,54 @@
+// exact-check
+
+// Checking that doc aliases are always listed after items with equivalent matching.
+
+const EXPECTED = [
+    {
+        'query': 'bar',
+        'others': [
+            {
+                'path': 'doc_alias_after_other_items',
+                'name': 'Bar',
+                'href': '../doc_alias_after_other_items/struct.Bar.html',
+            },
+            {
+                'path': 'doc_alias_after_other_items',
+                'name': 'foo',
+                'alias': 'Bar',
+                'is_alias': true
+            },
+        ],
+    },
```

Comment:
This test is ineffective, as `Bar` would lexicographically sort before `foo` anyways.  This could be fixed just by swapping the names, so the struct is `Foo` and the function is `bar`.

I ran the test against current master and it passed, which is generally never what you want for tests introduced alongside what they are testing

Reply:
Good point, updated the test.